src/System.IO/pkg/System.IO.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.IO.csproj">
-      <SupportedFramework>net463;netcoreapp1.1;uap10.1;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
oh is it because SystemException is not available in UWP? in that case should we just have a stub or something for now so that we don't drop support for a framework? cc: @weshaggard

---
In reply to: [82022610](https://github.com/dotnet/corefx/pull/12351#discussion_r82022610) [](ancestors = 82022610)